stoic.go

```diff
@@ -26,6 +35,17 @@ func main() {
 	openInEditor(dir + timeToFilename(time.Now()))
 }
 
+func version() string {
```

Comment:
Я бы переименовал и метод тоже
```suggestion
func about() string {
```

Reply:
поправил